index.rst

```diff
@@ -363,6 +363,19 @@ Cerro Pachon:
     --enable-foreman-plugin-dhcp-browser \
     --enable-foreman-proxy-plugin-remote-execution-ssh
 
+Foreman Console
+^^^^^^^^^^^^^^^
+Since we generated a self-signed certificate, you'll need to import it to the web browser you 
+are using - otherwise the remtote console won't be able.
```

Comment:
Two suggested fixes - one, type in `remote`, and two, indicate more specifically what can't connect.
```suggestion
are using - otherwise the Foreman remote console won't be able to connect.
```